packages/visualizations-react/package.json

```diff
@@ -54,7 +55,9 @@
     },
     "dependencies": {
         "@opendatasoft/visualizations": "0.27.4",
-        "use-callback-ref": "^1.2.4"
+        "use-callback-ref": "^1.2.4",
+        "react": "^17.0.2",
+        "react-dom": "^17.0.2"
```

Comment:
I don't think we want to package React in the lib itself 😬 Why did it move from devDependencies?

Reply:
It's just a try to see if it fixes test behavior in the platform (I tried other changes in the platform at the same time) 😅